tempesta_fw/http.c

```diff
@@ -2347,6 +2349,9 @@ tfw_http_resp_fwd(TfwHttpResp *resp)
 			TFW_DBG2("%s: Forwarding error: conn=[%p] resp=[%p]\n",
 				 __func__, cli_conn, req->resp);
 			BUG_ON(!req->resp);
+			spin_lock_bh(&cli_conn->seq_qlock);
+			list_del_init(&req->msg.seq_list);
+			spin_unlock_bh(&cli_conn->seq_qlock);
 			tfw_http_resp_pair_free(req);
```

Comment:
Do we need the loop if we deleted all the requests in the patch above? Maybe it makes sense to write 2 lines instead of 4?
```C
tfw_http_conn_msg_free(req->pair);
tfw_http_conn_req_clean(req);
```